core/domain/collection_services.py

```diff
@@ -949,23 +1184,39 @@ def get_next_page_of_all_non_private_commits(
 
 
 def _collection_rights_to_search_dict(rights):
-    # Allow searches like "is:featured".
+    """Allow searches like "is:featured".
+
+    Args:
+        rights: Collection rights of a collection
+    """
+
     doc = {}
     if rights.status == rights_manager.ACTIVITY_STATUS_PUBLICIZED:
         doc['is'] = 'featured'
     return doc
 
 
 def _should_index(collection):
+    """Checks if a particular collection should be indexed.
+
+    Args:
+        collection: Collection. The collection domain object to be checked if
```

Comment:
I think just `collection: Collection.` is fine. The sentence gets quite clumsy otherwise.